tools/cluster_features_and_label.py

```diff
@@ -20,7 +20,7 @@
 from vissl.utils.hydra_config import AttrDict, convert_to_attrdict, is_hydra_available
 from vissl.utils.io import save_file
 from vissl.utils.logger import setup_logging, shutdown_logging
-from vissl.utils.misc import merge_features, set_seeds, is_faiss_available
+from vissl.utils.misc import is_faiss_available, merge_features, set_seeds
```

Comment:
nit: is this change intended? or is this in master?

Reply:
@priyal, this looks like some changes autogenerated by the isort command. I am looking into it

Reply:
I think that is just `isort` fixing files that were previously non-conformant (alphabetical ordering of imports)